src/components/collection_voting_slab/MultiDatasetVoteSlab.js

```diff
@@ -0,0 +1,201 @@
+import {div, h} from "react-hyperscript-helpers";
+import CollectionSubmitVoteBox from "../collection_vote_box/CollectionSubmitVoteBox";
+import VotesPieChart from "../common/VotesPieChart";
+import VoteSummaryTable from "../vote_summary_table/VoteSummaryTable";
+import {filter, flatMap, flow, map, isNil, isEmpty, get, includes, every, forEach, groupBy, cloneDeep} from "lodash/fp";
+import {Storage} from "../../libs/storage";
+import {useEffect, useState} from "react";
+import DatasetsRequestedPanel from "./DatasetsRequestedPanel";
+import {dataUsePills} from "./ResearchProposalVoteSlab";
+
+const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    fontWeight: 'bold',
+    padding: '15px 25px',
+    margin: '10px 0 20px 0'
+  },
+  slabTitle: {
+    color: '#000000',
+    backgroundColor: '#F1EDE8',
+    fontSize: '1.6rem',
+    height: '32px',
+    width: 'fit-content',
+    padding: '1.5rem',
+    borderRadius: '4px 4px 0 0',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  dataUses: {
+    color: '#333F52',
+    backgroundColor: '#F1EDE8',
+    borderRadius: '0 4px 0 0',
+    padding: '15px 25px'
+  },
+  voteInfo: {
+    backgroundColor: '#D4E1EB',
+    padding: '22px 25px'
+  },
+  chairVoteInfo: {
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: '1.5rem'
+  }
+};
+
+
+export default function MultiDatasetVoteSlab(props) {
+  const [currentUserVotes, setCurrentUserVotes] = useState([]);
+  const [dacVotes, setDacVotes] = useState([]);
+  const [bucketDatasetIds, setBucketDatasetIds] = useState([]);
+  const {title, bucket, collectionDatasets, dacDatasetIds, isChair, isLoading} = props;
+
+
+  useEffect(() => {
+    const user = Storage.getCurrentUser();
+    const votes = !isNil(bucket) ? bucket.votes : [];
+
+    const dacVotes = flow(
+      map(voteData => voteData.dataAccess),
+      filter((dataAccessData) => !isEmpty(dataAccessData)),
+      map(filteredData => filteredData.memberVotes),
+      filter(memberVotes => includes(user.dacUserId, map(memberVote => memberVote.dacUserId)(memberVotes))),
+      flatMap(memberVotes => memberVotes)
+    )(votes);
+    setDacVotes(dacVotes);
```

Comment:
Idea for refactoring ... some of these flow blocks seem very re-usable. It might make sense to try and abstract these to utility methods like, `extractDacUserVotesfromBucket(user, bucket)`, etc. If you do, we should group them somewhere that is DarCollection specific for easier reference by subject/category/name, etc.